tests/console/nftables.pm

```diff
@@ -23,16 +23,23 @@ sub run {
 
     # check nftables, stop iptables and start firewalld by need
     my $logs = "/tmp/log";
+    my $dir  = "/etc/nftables";
 
-    #check that nftables has been already installed
+    # check that nftables has been already installed
     assert_script_run('echo -e "____ nftables version and rules ____\n\n" > ' . $logs);
-    assert_script_run("rpm -qa nftables |& tee -a $logs");
+    assert_script_run("zypper info nftables |& tee -a $logs");
     assert_script_run("if systemctl is-active -q iptables; then systemctl stop iptables; fi");
     systemctl("restart firewalld");
 
     # check that directory of nftables and create a customer rule 'tcp-mss-clamp'
     assert_script_run('echo -e "____ /etc/nftables contains rules ____\n" >> ' . $logs);
-    assert_script_run("ls /etc/nftables |& tee -a $logs");
+    # on Tumbleweed the reference directory is not created, so we need to add softfail for it.
```

Comment:
it's only referenced in a comment but again: Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ.

Reply:
The reference directory is missing on Tumbleweed, it is a bug.

Reply:
I understood that. I know that you consider it a bug. This is why you introduce the soft-fail. Still, if the bug persists then it will also affect other products that get the updated version.

Reply:
if factor first is followed, yes, but this changes (feature) comes from sles at first, so this is not the case then. So I think the bug is still valid.

Reply:
Not sure I understand. If a change hits SLES first, then why would Tumbleweed be affected?

But anyway, https://bugzilla.suse.com/show_bug.cgi?id=1183461#c8 is correct to state that one should not rely on that path when functionality is still ensured.

Reply:
okay, if the directory is not important then I won't check this at all. So I will remove related check, thanks.

Reply:
well, you are the test module maintainer. likely you added the check for a reason. I don't know what the reason was. Depends on that.